### PR TITLE
Feat: add alpha loading screen warning

### PR DESCRIPTION
**Pull request by craftablescience**
Here ya go @vos6434

- [x] Adds warning on load
- [x] Adds config option to disable the warning

Animation

### CODE REVIEW

File: src/main/resources/galacticraft.mixins.json [hunk 1]
```diff
@@ -32,7 +32,8 @@
     "client.InGameHudMixin",
     "client.PauseMenuScreenMixin",
     "client.PlayerInventoryScreenMixin",
-    "client.WorldRendererMixin"
+    "client.WorldRendererMixin",
+    "client.AlphaWarningTitleScreenMixin"
```
Comment (marcus8448) [inline src/main/resources/galacticraft.mixins.json:36]: Put this in alphabetical order

**Review by marcus8448 — CHANGES_REQUESTED** [review]

**Comment by JoeZwet** [comment]
Build failing isn't maven, it's license headers.

```
> License violations were found: src/main/java/dev/galacticraft/mod/mixin/client/AlphaWarningTitleScreenMixin.java
```

File: src/main/java/dev/galacticraft/mod/mixin/client/AlphaWarningTitleScreenMixin.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
+package dev.galacticraft.mod.mixin.client;
```
Comment (JoeZwet) [inline src/main/java/dev/galacticraft/mod/mixin/client/AlphaWarningTitleScreenMixin.java:1]: Missing license header.
```
> License violations were found: src/main/java/dev/galacticraft/mod/mixin/client/AlphaWarningTitleScreenMixin.java
```

**Review by JoeZwet — CHANGES_REQUESTED** [review]

File: src/main/resources/galacticraft.mixins.json (hunk 1 above)
Comment (craftablescience) [reply src/main/resources/galacticraft.mixins.json:36]: This is what I get for blindly following the MC mod dev Intellij plugin :/

File: src/main/java/dev/galacticraft/mod/mixin/client/AlphaWarningTitleScreenMixin.java (hunk 2 above)
Comment (craftablescience) [reply src/main/java/dev/galacticraft/mod/mixin/client/AlphaWarningTitleScreenMixin.java:1]: Woops, thanks

**Comment by craftablescience** [comment]
That's everything so far

**Review by marcus8448 — APPROVED** [review]

**Review by JoeZwet — APPROVED** [review]

**Pull request merged**